packages/ping-protect/tsconfig.lib.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../dist/out-tsc",
+    "declaration": true,
+    "types": ["node"]
```

Comment:
Probably don't need `node` types right? it may have come default but we probably dont need it since its browser only